### PR TITLE
MailService Integration

### PR DESCRIPTION
**Pull request by Jordex**

### CODE REVIEW

File: backend/src/main/java/de/tuberlin/enterprisecomputing/integrations/MailService.java [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
+    // declare request for employee 
+    SendEmailRequest employeeRequest = new SendEmailRequest()
+			.withDestination(new Destination().withToAddresses(new String[]{EMPLOYEE}))
+			.withMessage(new Message().withSubject(new Content().withData(this.employeeEmail()[0]))
+					.withBody(new Body().withText(new Content().withData(this.employeeEmail()[1]))));
+     
     public MailService(){
-        //TODO initialize SES client
+        //TODO initialize SES client, set region
+    	Region REGION = Region.getRegion(Regions.US_WEST_2);
```
Comment (maxim-tschumak) [inline backend/src/main/java/de/tuberlin/enterprisecomputing/integrations/MailService.java]: I think we have our Beanstalk in EU_WEST_1 (Ireland)

**Comment by maxim-tschumak** [comment]
Doesn't compile. Please check the dependencies in pom.xml. I think you forgot to include the AWS SDKs  dependencies. (for SES, S3 and DynamoDB)

**Comment by Jordex** [comment]
I added the dependencies (for SES, S3 and DynamoDB) manually. but when doing 'mvn clean package' no .jar file is created in the target file. Any idea what else could be wrong?

**Comment by maxim-tschumak** [comment]
You have to add the dependencies in the project's pom.xml. Maven will download them and add them to classpath then.

---- JonathanHeiss wrote ----

**Comment by maxim-tschumak** [comment]
looks good!

**Pull request merged**